cli/command/image/build.go

```diff
@@ -235,6 +247,28 @@ func runBuild(dockerCli *command.DockerCli, options buildOptions) error {
 		}
 	}
 
+	// replace Dockerfile if added dynamically
```

Comment:
If someone already has a Dockerfile  but its not being used for the build, and they use "-" on the `-f`, won't this replace their Dockerfile and cause incorrect results if their Dockerfile is used for some other purpose?